code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -627,18 +629,19 @@ func (fsh *StorageHandler) getReferencePath(ctx context.Context, r *http.Request
 
 func (fsh *StorageHandler) GetObjectPath(ctx context.Context, r *http.Request) (*blobberhttp.ObjectPathResult, error) {
 	allocationTx := ctx.Value(constants.ContextKeyAllocation).(string)
+	clientSign, _ := ctx.Value(constants.ContextKeyClientSignatureHeaderKey).(string)
+	ClientPublicKey, _ := ctx.Value(constants.ContextKeyClientKey).(string)
+	valid, err := verifySignatureFromRequest(allocationTx, clientSign, ClientPublicKey)
```

Comment:
Same as above. Also I think we can remove `GetObjectPath` endpoint. It is not required anywhere.

Reply:
Kishan is out for a week @lpoli, are you ok to make these changes to keep things moving?

Reply:
@stewartie4  I will be addressing the PR comments. Thanks.

Reply:
created a seperate PR for this. https://github.com/0chain/blobber/pull/863